src/amo/components/SearchForm/index.js

```diff
@@ -189,6 +189,10 @@ export class SearchFormBase extends React.Component {
     this.props.router.push(suggestion.url);
   }
 
+  shouldRenderSuggestions= (value) => {
```

Comment:
Nitpick but there should be a space before the `=` here, after `shouldRenderSuggestions`.